app/controllers/categories_controller.rb

```diff
@@ -11,4 +11,8 @@ def options
     res = res.find_all { |c| c[:text] =~ /\b#{Regexp.escape(params[:query].to_s)}/ }
     render json: res
   end
+
+  def index
+    render json: Category::CATEGORY_MAPPING.values
```

Comment:
prefer to put also keys. 
E.g.
```ruby
Category::CATEGORY_MAPPING.map { |k,v| v[:key] = k }
```

I know where to could I use it just now:)
uz ted vim, kde bych to pouzil:)